packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/__tests__/quanticHeadlessLoader.test.js

```diff
@@ -565,10 +567,83 @@ describe('c/quanticHeadlessLoader', () => {
         state: {someFacets: {label: 'label', facetId: 'facetId'}},
       });
       const data = getFromStore(testId, 'someFacets');
-
       expect(data).toBeDefined();
       expect(data.label).toBe('label');
       expect(data.facetId).toBe('facetId');
     });
   });
+
+  describe('registerSortOptionsToStore', () => {
+    beforeEach(() => {
+      window.coveoHeadless = {
+        [testId]: {
+          bindings: {},
+        },
+      };
+    });
+
+    it('should register the sort options data in the store', () => {
+      initQuanticStoreTest(testId, {state: {someSortOptions: []}});
+      registerSortOptionsToStore(testId, [
+        {label: '', value: '', criterion: {by: 'relevancy'}},
+      ]);
+
+      expect(
+        window.coveoHeadless[testId].bindings.store.state.sort
+      ).toBeDefined();
+      expect(mockedConsoleError.mock.calls.length).toBe(0);
+    });
```

Comment:
we need to test that the value you registered, was indeed saved in the store, this is the essence of this test.